app/models/alert_types/untrained_students_alert.rb

```diff
@@ -20,7 +20,7 @@
 #  details        :text(65535)
 #
 
-# Alert for a course that has no enrolled students after it is underway
+# Receive an alert when a course has no enrolled students even after it has started.
```

Comment:
This is also incorrect. This kind of Alert indicates that a significant portion of enrolled students have not completed their assigned training modules.

Reply:
Can you explain me little bit . what is should be?

Reply:
It should be something like 'Alert for when the students in a course are behind schedule on their assigned training modules.'